From foo@baz Wed Jul 29 09:05:33 PDT 2009
Date: Wed, 29 Jul 2009 09:05:33 -0700
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: Staging: hv: fix sparse function warnings

From: Greg Kroah-Hartman <gregkh@suse.de>

This fixes up all of the sparse warnings about functions not being
properly declared.  Meaning, void functions need to say they are a void
function, otherwise the compiler assumes it is an integer here.

Cc: Hank Janssen <hjanssen@microsoft.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/hv/Connection.c |    3 +--
 drivers/staging/hv/Hv.c         |    3 +--
 2 files changed, 2 insertions(+), 4 deletions(-)

--- a/drivers/staging/hv/Connection.c
+++ b/drivers/staging/hv/Connection.c
@@ -45,8 +45,7 @@ Description:
 
 --*/
 static int
-VmbusConnect(
-	)
+VmbusConnect(void)
 {
 	int ret=0;
 	VMBUS_CHANNEL_MSGINFO *msgInfo=NULL;
--- a/drivers/staging/hv/Hv.c
+++ b/drivers/staging/hv/Hv.c
@@ -445,8 +445,7 @@ Description:
 
 --*/
 static HV_STATUS
-HvSignalEvent(
-	)
+HvSignalEvent(void)
 {
 	HV_STATUS status;
 
